addon/components/mediasuite-radio-buttons-boolean.js

```diff
@@ -1,30 +1,29 @@
 import Ember from 'ember'
-const { computed } = Ember
+const { computed, get } = Ember
 
 export default Ember.Component.extend({
   name: '',
-  value: false,
-  options: computed('name', 'value', 'disabled', function () {
+  computedValue: computed('value', 'options', function () {
+    const value = get(this, 'value')
+    return value == null ? null : get(this, 'options').find(option => option.value === value)
```

Comment:
`.find(option => get(option, 'value') === value)`

Don't assume this is a POJO